.github/workflows/prerelease.yml

```diff
@@ -2,9 +2,6 @@ name: prerelease
 
 on:
   pull_request:
-    paths:
-    - '.github/workflows/prerelease.yml'
```

Comment:
Why are we deleting the paths here? Isn't it still useful to trigger the run when prerelease.yml changes?

Reply:
The workflow might  erase the testing podspecs repo and push all podspecs. This process takes three hours and  might fail presubmit tests.